client-install-raspberry-pi.sh

```diff
@@ -6,9 +6,7 @@
 VERSION=1.0.0
 CONFIG=/boot/config.txt
 NINENINERULES=/etc/udev/rules.d/99-com.rules
-INSTALL_DIRECTORY=~/
-FOLDER_NAME=stream-pi-client/
-
+INSTALL_DIRECTORY="$HOME/stream-pi-client"
```

Comment:
The decision to keep the install directory and folder name different was on purpose.

Moreover, if you look at your new script, you can see that it actually first unzips perfectly, (line 55) and then deletes the install directory itself (line 62).

Reply:
Right, I overlooked this one -- will update.